influxdb/__init__.py

```diff
@@ -0,0 +1,148 @@
+
+
+#!/usr/bin/env python3
```

Comment:
I don't know if these emptly line is a problem, but could be one with the shebang.